README.md

```diff
@@ -6,9 +6,7 @@
 
 ## Screenshots:
 
-### mobile
-
```

Comment:
- [ ] **[OPTIONAL]** It would be nice to fix your typo error in the documentation (see image). I don't think there is a count method. And I think you meant `npm test` to run the test, not `npm start`. Or you could delete the line if you don't mind.

  ![ranjeet](https://user-images.githubusercontent.com/36542986/148215145-fbf28be5-c8c8-43a2-8bc7-497a475f57c9.jpg)

Reply:
> * [ ]  **[OPTIONAL]** It would be nice to fix your typo error in the documentation (see image). I don't think there is a count method. And I think you meant `npm test` to run the test, not `npm start`. Or you could delete the line if you don't mind.
>   ![ranjeet](https://user-images.githubusercontent.com/36542986/148215145-fbf28be5-c8c8-43a2-8bc7-497a475f57c9.jpg)

Hello @TedLivist ,
Thank you for your review, And yes this is a typo in my Readme file. Thank you for pointing it actually i copied my last project Readme content into this one, And i missed this line
I will update it now 
Once again thank you

Reply:
Alright @thecodechaser you're welcome